example/widget_surveyor.dart

```diff
@@ -160,6 +160,8 @@ class AnalysisResult {
       {'name': appName, 'widgets': widgetReferences};
 }
 
+// bug: fixed in linter 0.1.116 (remove once landed)
+// ignore: prefer_mixin
```

Comment:
Was this moved by the sort feature? If so, we really need to fix it to keep preceding comments with a declaration.

Reply:
Yep.  Sorting and formatting both wreak havoc on non-doc comments.

Reply:
I'm not aware of any problems with formatting, only with sorting. If you can reproduce this we should let Bob know.

Reply:
Whoops, I misspoke. I think the formatter is OK.  Sorry for the noise!